eng/Dependencies.props

```diff
@@ -29,7 +29,6 @@ and are generated based on the last package release.
     <LatestPackageReference Include="Microsoft.CodeAnalysis.Razor" Version="$(MicrosoftCodeAnalysisRazorPackageVersion)" />
     <LatestPackageReference Include="Microsoft.CSharp" Version="$(MicrosoftCSharpPackageVersion)" />
     <LatestPackageReference Include="Microsoft.DotNet.GenAPI" Version="$(MicrosoftDotNetGenApiPackageVersion)" />
-    <LatestPackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="$(MicrosoftDotNetPlatformAbstractionsPackageVersion)" />
```

Comment:
Should also remove the entry in Versions.props and Versions.Details.xml.

Reply:
Hmm... nvm this package is special in that it doesn't exist elsewhere. Strange.

Reply:
Yeah, I was surprised that this was the only reference to PlatformAbstractions in the whole repo. Maybe it was a hold over from previous versions when aspnet actually did reference it. But since 3.0, aspnet hasn't used it.